src/c-writer.cc

```diff
@@ -1200,6 +1344,9 @@ void CWriter::WriteExports(WriteExportsKind kind) {
 void CWriter::WriteInit() {
   Write(Newline(), "void ", module_prefix_, "_init(void) ", OpenBrace());
   Write("init_func_types();", Newline());
+  if (!module_->tags.empty()) {
+    Write("init_tags();", Newline());
+  }
```

Comment:
Given the other lines here don't check for empty perhaps we should do this optization as a followup and apply it to all of these init functions?

Reply:
Done.